packages/suite/src/views/settings/debug/Processes.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState, useMemo } from 'react';
+import { Checkbox } from '@trezor/components';
+import { desktopApi } from '@trezor/suite-desktop-api';
+
+import { ActionColumn, SectionItem, TextColumn } from '@suite-components/Settings';
+
+interface Process {
+    service: boolean;
+    process: boolean;
+}
+
+export const Processes = () => {
+    const [bridgeProcess, setBridgeProcess] = useState<Process>({ service: false, process: false });
+
+    const items = useMemo(() => {
+        return [
+            {
+                name: 'Bridge',
+                onToggle: () => desktopApi.toggleBridge(),
+                isChecked: bridgeProcess.process === true,
+            },
+        ];
+    }, [desktopApi, bridgeProcess]);
+
+    useEffect(() => {
+        desktopApi.getBridgeStatus().then(result => {
+            if (result.success) {
+                setBridgeProcess(result.status);
+            }
+        });
+
+        const onBridgeProcessEvent = (status: Process) => {
+            setBridgeProcess(status);
+        };
+        desktopApi.on('bridge/status', onBridgeProcessEvent);
+
+        // todo: how to remove event listener from desktopApi?
+        // return () => {
+        //     desktopApi.remove('bridge/status', onBridgeProcessEvent);
+        // };
```

Comment:
Currently only this, unfortunately, but should be ok here.
```suggestion
        return () => {
            desktopApi.removeAllListeners('bridge/status');
        };
```

Reply:
top af9f722aed67c3a268995e340867bd9d37752c3d